components/eamxx/src/physics/shoc/tests/infra/shoc_ic_cases.cpp

```diff
@@ -16,11 +16,10 @@ namespace {
 // top and then flips everything, so we do the same.
 //------------------------------------------------------------------------
 
-using KT     = KokkosTypes<HostDevice>;
 using Scalar = Real;
-using Array1 = typename KT::template lview<Scalar*>;
-using Array2 = typename KT::template lview<Scalar**>;
-using Array3 = typename KT::template lview<Scalar***>;
+using Array1 = typename FortranData::Array1;
```

Comment:
What's the meaning of "FortranData" here? Should this be maybe called something like "BaselineData" or is there still some Fortran reminescence?

Reply:
@bartgol , I made note of those in the follow-on work. These will be renamed but i don't think it's worth doing another CI run on this PR.